pkg/operator/kube_cloud_config/transform.go

```diff
@@ -0,0 +1,99 @@
+package kube_cloud_config
```

Comment:
I would think about splitting this file to 
bootstrap.go => has the handler for bootstrap transforming 
moving general transform function like `CloudConfigTransformers` to controller.go

Reply:
I should probably also write up some tests here.